### PR TITLE
 fix(Scripts/Spell): fix Spinning Up — Mimiron phase4 

### LINKED ISSUE #1611: [Ulduar] Spinning Up Mimiron P4  [$90]
[Ulduar] Spinning Up Mimiron P4 

##### SMALL DESCRIPTION:
Mimiron Spinning up spell in P4 doesn't work

##### EXPECTED BLIZZLIKE BEHAVIOUR:
When Mimiron arrive in P4 , the VX-001 have to use the spell Spinning-Up like in the P2

##### CURRENT BEHAVIOUR:
Actually , VX-001 use the Spinning up only on P2 and not on P4 , and if we uncomment this line : https://github.com/azerothcore/azerothcore-wotlk/blob/03bfbf4f7e43f6b8064e6846e0bf2503958c17bc/src/server/scripts/Northrend/Ulduar/Ulduar/boss_mimiron.cpp#L1376 , VX-001 use the spell , but visual is not correct with damage done by the spell ...

##### STEPS TO REPRODUCE THE PROBLEM:

1.  Enter Ulduar 10 or 25 players
2. Travel to Mimiron and engage him
3. See the VX-001 P2 is correct but not in P4

##### BRANCH(ES):

master

##### OPERATING SYSTEM: Debian 9

<bountysource-plugin>

---
There is a **[$90 open bounty](https://www.bountysource.com/issues/71801325-ulduar-spinning-up-mimiron-p4?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** on this issue. Add to the bounty at [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Viste**
##### CHANGES PROPOSED:
-  possible fix for visual part of spell Spinning Up on phase 4 Mimiron encounter
##### HOW TO TEST THE CHANGES:
    Enter Ulduar 10 or 25 players
    Travel to Mimiron and engage him
    See the VX-001 P2 and  P4

##### Target branch(es): Master

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1553390628412350005.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1553390628412350005');
+
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES (63414, 'spell_mimiron_spinning_up');
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1553390628412350005.sql]: - please always use `DELETE` before `INSERT`
- missing new line at the end of file

**Comment by pklloveyou** [comment]
P4 fire orientation error insanity
Sorry translation software

**Comment by Viste** [comment]
@pklloveyou what?

**Comment by Naxxramas-Server** [comment]
No that's same problem , not work , because i think , when the 3 parts of the boss are glued , VX-001 is never in front and is always offbeat and wrong angle.
I think you have to rework the Positions on each part for the P4.

**Comment by pklloveyou** [comment]
Wrong direction and Angle P4

**Comment by Viste** [comment]
@Naxxramas-Server Okay, thanks for the clear and understandable answer.
@pklloveyou Please use deepl.com (it's a good translator) or don't write anything at all, because it's not clear from your first comment at all.

**Comment by FrancescoBorzi** [comment]
even if it's not clear, it's fine that people try to help as long as the intentions are good

**Pull request closed without merging**